maxentep/utils.py

```diff
@@ -2,6 +2,7 @@
 import scipy.stats as ss
 import matplotlib.pyplot as plt
 import maxentep
+import tensorflow as tf
```

Comment:
Unused?

Reply:
It is used in `compartment_restrainer` and  `get_dist` functions. I will add the `import` to the functions.

Reply:
It's fine, I just didn't notice the code that used it. Keep it in there.